system/include/emscripten/fetch.h

```diff
@@ -23,33 +23,32 @@ extern "C" {
 
 // If passed, the intermediate streamed bytes will be passed in to the
 // onprogress() handler. If not specified, the onprogress() handler will still
-// be called, but without data bytes.  Note: Firefox only as it depends on
-// 'moz-chunked-arraybuffer'.
+// be called, but without data bytes.
 #define EMSCRIPTEN_FETCH_STREAM_DATA 2
 
-// If passed, the final download will be stored in IndexedDB. If not specified,
+// If passed, the final download will be stored in browser cache. If not specified,
```

Comment:
I don't think we should ditch the IndexDB stuff as part of this PR.. can we keep around for now. We can consider removing it as a followup perhaps.